portsyncd/portsyncd_helper.cpp

```diff
@@ -0,0 +1,64 @@
+#include "portsyncd/linksync.h"
```

Comment:
I think we discussed it last time, please move this to portsyncd itself. I don't see a reason why it has to be part of helper file and cause merge conflicts for future bug fixes.